recipe/meta.yaml

```diff
@@ -1,63 +1,92 @@
-{% set version = "0.9.0" %}
+{% set name = "geopandas" %}
+{% set version = "0.12.2" %}
 
 package:
-  name: geopandas
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/geopandas/geopandas/releases/download/v{{ version }}/geopandas-{{ version }}.tar.gz
-  sha256: 63972ab4dc44c4029f340600dcb83264eb8132dd22b104da0b654bef7f42630a
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0acdacddefa176525e4da6d9aeeece225da26055c4becdc6e97cf40fa97c27f4
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  skip: True  # [py<38]
 
 outputs:
   - name: geopandas-base
-    script: build_base.sh
-    build:
-      noarch: python
+    script: build_base.sh   # [unix]
+    script: build_base.bat  # [win]
     requirements:
       host:
-        - python >=3.6
+        - python
         - pip
+        - setuptools
+        - wheel
       run:
-        - python >=3.6
-        - pandas >=0.24
-        - shapely
-        - pyproj >=2.2.0
+        - python
+        - packaging
+        - pandas >=1.0.5
+        - pyproj >=2.6.1.post1
+        - shapely >=1.7
 
     test:
       imports:
         - geopandas
         - geopandas.io
+        - geopandas.tools
+        - geopandas.datasets
       files:
         - test_data
 
   - name: geopandas
     requirements:
       host:
-        - python >=3.6
+        - python
+        - pip
+        - setuptools
+        - wheel
       run:
-        - python >=3.6
+        - python
         - {{ pin_subpackage('geopandas-base', exact=True) }}
-        - fiona
-        - rtree
+        - fiona >=1.8
+        # fiona package isn't available yet on the defaults channel.
```

Comment:
Fiona is now on defaults (now including Python 3.10) for all but s390x, so you should be able to build this now.

Reply:
Thanks Melody!